src/Microsoft.ML.LightGBM/LightGbmBinaryTrainer.cs

```diff
@@ -11,7 +11,7 @@
 using Microsoft.ML.Runtime.LightGBM;
 using Microsoft.ML.Runtime.Model;
 
-[assembly: LoadableClass(LightGbmBinaryTrainer.Summary, typeof(LightGbmBinaryTrainer), typeof(LightGbmArguments),
+[assembly: LoadableClass(LightGbmBinaryTrainer.UserName, typeof(LightGbmBinaryTrainer), typeof(LightGbmArguments),
```

Comment:
>UserName [](start = 47, length = 8)

The first thing I believe *is* the summary, or should be. If it was badly written, that's fine, we ought to improve it, but to use the user-name as the first argument is wrong.

Also the thing that is *actually* being filled in as the user name is a string literal (that somehow differs from the field declared as `UserName`.